### DISCUSSION TITLE
how to query peering in a VRF

### DISCUSSION THREAD

**Discussion by MEYERcyril** [peering-manager/peering-manager#774] [Q&A]
Hi,
We have deployed and played a bit with peering manager and are stucked.
Our Network is composed of multiple VRF and the peerings are in a VRF called : INTERNET.
The tools do not help to provide this information anywhere and include it in the request to the router.
Maybe somebody went thru this and can help to figure it out.

Best regrads.

**Comment by netravnen** [comment]
@MEYERcyril can you showcase (screenshots, which URLs) _where_ in the software you are experiencing this?

From your description. I assume you mean you want to see the VRF name in the BGP session overview under both your ASN and under the router device. Which is not currently possible. (The peer list is available, without VRF information)

Correct?

**Reply by MEYERcyril** [reply]
Hi, 
We don't see the peer list as it is inside the VRF. it can't be querried. 
We use IOS-XR routers. 
Simplified conf
router bgp 18200
vrf INTERNET
neighbour X.x.x.x

We already have quite a lot of peering sessions that are established.
nothing appears in tabs : connections and direct peering connections in the router section.

Thanks for advises

**Reply by MEYERcyril** [reply]
Also, where can I find the file for modifying the platform template.

**Reply by gmazoyer** [reply]
Not sure what you are trying to accomplish. You won't see any BGP sessions in Peering Manager until you create them in the software yourself. It cannot discover BGP sessions on your device except for BGP sessions over IXPs for IXPs that are imported using PeeringDB.

Platform has not template. If you are speaking about changing the platform model for your own need, [this is the file you are looking](https://github.com/peering-manager/peering-manager/blob/main/devices/models.py).
